### ISSUE TITLE
Map Editor Issues

### ISSUE THREAD

**Issue by AdamDN** [SmartlyDressedGames/Unturned-3.x-Community#2667]
Hello, I haven't touched the map Editor in a couple of years, so if I'm not sure if this is intentional. 
But why are there no maps showing up in the Editor, even maps that are newly created? They can be created, edited, and saved - and then they do not appear on the editor list. 
![image](https://user-images.githubusercontent.com/71527679/122652955-820a8280-d139-11eb-8830-5888b3a813b0.png)

**Comment by SDGNelson** [comment]
Can you post the Client.log file after that happens please?

**Comment by AdamDN** [comment]
Attached. 
I opened the game, created a new map, clicked Edit, saved the map, exited, and then the map is no longer listed in the editor. 
I can see the map files in file explorer, and it also prevents me from creating another a new map with the same name, but the map is not in the list. 

[Client.log](https://github.com/SmartlyDressedGames/Unturned-3.x-Community/files/6681318/Client.log)

**Comment by SDGNelson** [comment]
Thanks for the log! This line seems like a likely candidate, but apparently RI.Hid is raw-input related:
```
[2021-06-19 21:31:23] <RI.Hid> Failed to create device file:
2 The system cannot find the file specified.
```
The level loading code is pretty old and bad, so it would not surprise me if there is a bug to find here. If you create a file named ".unlocker" in your level's folder does it show up in the list? e.g. Unturned/Maps/PEI/.unlocker

**Comment by SDGNelson** [comment]
Did you find a workaround? What ended up being the problem?

**Comment by AdamDN** [comment]
Oops, my mistake. I meant to give a follow-up before closing it. 
The .unlocker did not help, I tried this before opening the issue. 

I tried many things, and I'm not exactly sure which one was the fix.
I copy pasted the files from PEI into my map folder, and then deleted the .config file. Which seemed to be a repeatable fix.

I managed to get another map working without cloning PEI files, but I'm unsure what worked for that.
I believe that the fix was adding a blank config.json file into the folder, but I also messed around with deleting/regenerating Editor files which may have fixed it.
